components/modal/subscriptions-modal/dataset-manager/component.js

```diff
@@ -142,7 +142,7 @@ class DatasetManager extends Component {
       <div className="c-dataset-manager">
         {selectedDatasets.map((_selectedDataset, index) => (
           <div
-            key={_selectedDataset.id}
+            key={index}
```

Comment:
Usually, React doesn't recommend using indexes as keys, wasn't `_selectedDataset.id` value working properly? 🤔

Reply:
Ops, sorry, didn't read your comments in the PR regarding this. Still, use indexes, maybe we can try the combination of different values to make it unique, what do you think?

Reply:
@andresgnlez 
Yes, you right. This is not optimal.
Possible solutions that I see are:
1. use the **index** in some combination with **id**
2. generate unique id on adding new items
I think this is not a big deal :smile: Agree?

Reply:
I think second one is the way to go, first one eslint will still complain because you are using the index variable 😛

Reply:
I have tried a way to add a unique ID for each dataset. But this will be a complex solution for a minor problem and hard to catch in future modifications.

> When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort. (reactjs.org)